src/app/recurring-deposits/recurring-deposits-account-view/recurring-deposits-account-view.component.html

```diff
@@ -0,0 +1,202 @@
+<mat-card class="recurringDeposits-account-card">
+
+  <mat-card-header fxLayout="column" class="header">
+
+    <mat-card-title-group class="header-title-group">
+
+      <div class="profile-image-container">
+        <div>
+          <img mat-card-md-image class="profile-image" matTooltip="Recurring Deposits Account"
+            [src]="'assets/images/recurring-deposits_account_placeholder.png'">
+        </div>
+      </div>
+
+      <div class="mat-typography account-card-title">
+        <mat-card-title>
+          <h3>
+            <i class="fa fa-stop" [ngClass]="recurringDepositsAccountData.status.code | statusLookup"
+              [matTooltip]="recurringDepositsAccountData.status.value"></i>
+            Account Name : {{recurringDepositsAccountData.depositProductName}}
+          </h3>
+        </mat-card-title>
+        <mat-card-subtitle>
+          <p>
+            Account #: {{recurringDepositsAccountData.accountNo}} | Client Name:
+            {{recurringDepositsAccountData.clientName}}<br />
+            <span
+              *ngIf="!recurringDepositsAccountData.status.rejected && !recurringDepositsAccountData.status.submittedAndPendingApproval">
+              Current Balance:
+              {{recurringDepositsAccountData.currency.displaySymbol}}&nbsp;{{recurringDepositsAccountData.summary.accountBalance}}<br />
+              Available Balance:
+              {{recurringDepositsAccountData.currency.displaySymbol}}&nbsp;{{recurringDepositsAccountData.summary.availableBalance}}<br />
+            </span>
+          </p>
+        </mat-card-subtitle>
+      </div>
+
+    </mat-card-title-group>
+
+    <mat-card-actions class="account-actions">
+
+      <ng-container *ngFor="let button of buttonConfig.singleButtons">
+        <button mat-raised-button >
+          <i class="{{button.icon}}"></i> {{button.name}}</button>
+      </ng-container>
+  
+      <ng-container *ngIf="buttonConfig.options.length > 0">
+        <button mat-raised-button [matMenuTriggerFor]="More">More</button>
+        <mat-menu #More="matMenu">
+        <span *ngFor="let option of buttonConfig.options">
+          <button mat-menu-item >{{option.name}}</button>
+        </span>
+        </mat-menu>
+      </ng-container>
+
+    </mat-card-actions>
+
+  </mat-card-header>
+
+  <mat-card-content class="content">
+
+    <div class="recurringDeposits-account-tables" fxLayout="row" fxLayoutGap="2%">
+
+      <div fxFlex="49%"
+        *ngIf="!recurringDepositsAccountData.status.rejected && !recurringDepositsAccountData.status.submittedAndPendingApproval">
+        <h4 class="table-headers">Savings Details</h4>
```

Comment:
reccuring deposits details

Reply:
Please have a look at the community app code.